### DISCUSSION TITLE
What is the exactly differences between StyledProperty.SetValue() and StyledProperty.SetCurrentValue()?

### DISCUSSION THREAD

**Discussion by AlvinRey** [AvaloniaUI/Avalonia#15361] [Q&A]
I had watched the summary about StyledProperty.SetValue() and StyledProperty.SetCurrentValue() in the sorce code(AvaloniaObject.cs), but its too abstract to me. Can anyone give some examples to explain it for me? Thank for any help!!

**Comment by maxkatz6** [comment]
This answer explains it well https://stackoverflow.com/a/4232379
